### PR TITLE
Bundle examples

### PR DESCRIPTION
**Pull request by nielslyngsoe**
## Description

Added examples for extension type of Bundle

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

**Comment by eshanrnh** [comment]
Thanks for the PR, @nielslyngsoe 🙌 We will review it as soon as we can.

File: 14/umbraco-cms/extending/backoffice-setup/extension-registry/bundle.md [hunk 1]
```diff
@@ -8,4 +8,46 @@ The `bundle` extension type enables you to gather many extension manifests into
 
 It points to a single JavaScript file that via JavaScript Modules, exports or re-exports Extension Manifests written in JavaScript.
 
-This means that it can be used as the entry point for a package.
+It can be used as the entry point for a package, or as a grouping for a set of manifests.
+
+## Use Bundle as entry point for a package
+
+If you like to declare your manifests in JavaScript/TypeScript, the Bundle is a great choice for you.
```
Comment (jonat123) [inline 14/umbraco-cms/extending/backoffice-setup/extension-registry/bundle.md:15]:
```suggestion
If you want to declare your manifests in JavaScript/TypeScript, the Bundle is a great choice.
```

File: 14/umbraco-cms/extending/backoffice-setup/extension-registry/bundle.md [hunk 2]
```diff
@@ -8,4 +8,46 @@ The `bundle` extension type enables you to gather many extension manifests into
...
+
+The following example shows a umbraco-package that referees to one bundle, which then can declare several manifests.
```
Comment (jonat123) [inline 14/umbraco-cms/extending/backoffice-setup/extension-registry/bundle.md:17]:
```suggestion
The following example shows an umbraco-package that referees to one bundle, which can then declare several manifests.
```

**Review by jonat123 — APPROVED** [review]
@nielslyngsoe Thanks for adding the suggestions 😄

The PR is approved and I'll merge it in 😄

**Pull request merged**